modules/apps/callbacks/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=ibc-go-callbacks
```

Comment:
I don't have any experience with this particular tool, so I'm happy to trust you with these configs and if there are any issues we can worry about it in a follow up :muscle:

Reply:
I took the SDK as reference, so I hope it is correct. :)